akka-actor-tests/src/test/scala/akka/io/TcpConnectionSpec.scala

```diff
@@ -364,7 +364,7 @@ class TcpConnectionSpec extends AkkaSpec("""
         ConfigFactory.load(
           ConfigFactory.parseString("akka.io.tcp.direct-buffer-size = 1k")
             .withFallback(AkkaSpec.testConf))
-      override implicit def system: ActorSystem = ActorSystem("respectPullModeTest", config)
+      override lazy val system: ActorSystem = ActorSystem("respectPullModeTest", config)
```

Comment:
This test started failing after this change, but previously it created half a dozen actor systems. Can someone that is familiar with this test help me debug it. You will see the failure in the jenkins log (or checkout the branch).